ModulemdTranslationHelpers/Utils.py

```diff
@@ -1,14 +1,97 @@
 import sys
+import logging
+import gi
 from babel.messages import Catalog, pofile
 from datetime import datetime
 from collections import defaultdict
 
-try:
-    import gi
-    gi.require_version('Modulemd', '2.0')
-    from gi.repository import Modulemd
-except ImportError:
-    sys.exit(77)
+gi.require_version('Modulemd', '2.0')
```

Comment:
```suggestion
gi.require_version('Modulemd', '2.0')  # noqa
```
(Note that it must be over by two spaces from the end of the line or pycodestyle will complain).